File: mpd-concursos-app-frontend/src/app/models/jwt-dto.ts
export class JwtDto {
    token: string;
    type: string;
    username: string;
    authorities: { authority: string }[];

    constructor(token: string = '', type: string = 'Bearer', username: string = '', authorities: { authority: string }[] = []) {
        this.token = token;
        this.type = type;
        this.username = username;
        this.authorities = authorities;
    }
} 

File: concurso-backend/src/main/java/ar/gov/mpd/concursobackend/auth/application/dto/JwtDto.java
package ar.gov.mpd.concursobackend.auth.application.dto;

import lombok.Data;
import org.springframework.security.core.GrantedAuthority;

import java.util.Collection;

@Data
public class JwtDto {

    private String token;
    private String bearer = "Bearer";
    private String username;
    private Collection<? extends GrantedAuthority> authorities;
    private String cuit;

    public JwtDto() {}
    
    public JwtDto(String token, String username, Collection<? extends GrantedAuthority> authorities, String cuit) {
        this.token = token;
        this.username = username;
        this.authorities = authorities;
        this.cuit = cuit;

    }

}
